t/02-bowels/80-repo-lock.t

```diff
@@ -113,4 +114,29 @@ my $t = Pinto::Tester->new;
 }
 
 #------------------------------------------------------------------------------
+subtest 'Test stale lock file' => sub {
+
+    # create dummy lock file not connected to us
+    my $lockfile = $t->root->file('.lock');
+    my $fh = IO::File->new( $lockfile, 'w' );
```

Comment:
You could also just say `$lockfile->openw;` to open the file for writing (see the perldoc for `Path::Class::File`).

Or is there a particular reason for using `IO::File`? If so, then we also need to `use IO::File` in this test.